examples/sample-pub/views/article.html

```diff
@@ -70,7 +70,7 @@
          }
       },
       "description": "A most wonderful article",
-      "isAccessibleForFree": "False",
+      "isAccessibleForFree": "True",
```

Comment:
Perhaps make a separate example for this rather than changing the default one?

Reply:
Sorry, I forgot that I could set a .gitignore file and accidentally committed changes to those files I made for test purposes only. I've reset them back to the correct versions.